### PR TITLE
Add missing labels for Account Console (v3)

### LINKED ISSUE #21805: Missing labels account console
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

account/ui

### Describe the bug

There are some missing labels on the account console:

image

### Version

main

### Expected behavior

translate

### Actual behavior

not translated

### How to Reproduce?

go to the application page of the account ui and have an client with consent turned on

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #21805

### CODE REVIEW

File: js/apps/account-ui/public/locales/ar/translation.json
Comment (jonkoops) [inline js/apps/account-ui/public/locales/ar/translation.json:1]: @Eng-Fouad can I ask you to verify the translations here are correct?

File: js/apps/account-ui/public/locales/en/translation.json [hunk 1]
```diff
@@ -97,7 +98,7 @@
   "removeConsentError": "Could not remove consent due to: {{error}}",
   "removeConsentSuccess": "Successfully removed consent",
   "removeCred": "Remove {{0}}",
-  "removeModalMessage": "Hiermee wordt de momenteel verleende toegangsrechten voor {{0}} verwijderd. U moet opnieuw toegang verlenen als u deze app wilt gebruiken.",
+  "removeModalMessage": "This will remove the currently granted access permission for {{0}}. You will need to grant access again if you want to use this app.",
```
Comment (jonkoops) [inline js/apps/account-ui/public/locales/en/translation.json:101]: Looks like I accidentally changed this label to Dutch in a previous PR, so I've fixed it here.

File: js/apps/account-ui/public/locales/fa/translation.json
Comment (jonkoops) [inline js/apps/account-ui/public/locales/fa/translation.json:1]: @sharifrezvani can I ask you to verify the translations here are correct?

**Review by edewit — APPROVED** [review]

**Comment by jonkoops** [comment]
I am going ahead and merge this in, if any of the translations are off they can be fixed in another PR.

File: js/apps/account-ui/public/locales/ar/translation.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   "client": "العميل",
   "clients": "العملاء",
   "close": "إغلاق",
+  "confirm": "يتأكد",
```
Comment (Eng-Fouad) [inline js/apps/account-ui/public/locales/ar/translation.json:21]: It should be `تأكيد`. I will create a new PR.

Comment (Eng-Fouad) [reply js/apps/account-ui/public/locales/ar/translation.json:21]: Here it is #21813

**Review by sharifrezvani — COMMENTED** [review]
Sure thing. Sorry for the late reply. Since this is already merged, I will not go into details about translation issues. The changes here are good enough for now and they won't be an issue. I will open up another PR in order to correct the mistakes and add further translations later

**Pull request merged**